### PR TITLE
Migrates to One Call 3.0 API

### PR DESCRIPTION
**Pull request by jeanmartin**
This doesn't take full advantage of all new One Call 3.0 features, but it adds the `summary` field to the `daily` response.

Addresses #37

### CODE REVIEW

**Comment by dblock** [comment]
Thanks!

Update CHANGELOG? Not sure why Danger didn't pick it up 🤔 
https://github.com/dblock/open-weather-ruby-client/blob/master/CHANGELOG.md

Increment version to 0.5.0 in this PR (version.rb, CHANGELOG, etc.), this is a major version change that breaks things.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-### 0.4.1 (Next)
+### 0.5.0 (Next)
 
 * [#25](https://github.com/dblock/open-weather-ruby-client/pull/25): Exposed the national weather alerts response in the One Call API - [@troya2](https://github.com/troya2).
-* Your contribution here.
+* [#38](https://github.com/dblock/open-weather-ruby-client/pull/38): Migrates to One Call 3.0 API - [@jeanmartin](https://github.com/jeanmartin).
+* Your contribution here
```
Comment (dblock) [inline CHANGELOG.md:5]: Nit: add a period, danger complains.

**Review by dblock — CHANGES_REQUESTED** [review]
Danger is unhappy. Almost there. Thanks for hanging in here with me!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
-### 0.4.1 (Next)
+### 0.5.0 (Next)
 
 * [#25](https://github.com/dblock/open-weather-ruby-client/pull/25): Exposed the national weather alerts response in the One Call API - [@troya2](https://github.com/troya2).
-* Your contribution here.
+* [#38](https://github.com/dblock/open-weather-ruby-client/pull/38): Migrates to One Call 3.0 API - [@jeanmartin](https://github.com/jeanmartin).
```
Comment (dblock) [inline CHANGELOG.md:4]: Let's say "Migrated" to be consistent with the rest of the log.

**Comment by jeanmartin** [comment]
No problem, Dangers happiness is important to me :D

**Review by dblock — APPROVED** [review]

**Comment by dblock** [comment]
Thanks, merged. I'll cut a release sometime sooner than later.

**Comment by dblock** [comment]
With this change I am seeing this in some logs:

```
WARN -- : Error getting weather at 50.06789, 19.92789 on 1727631664 for  ... Please note that using One Call 3.0 requires a separate subscription to the One Call by Call plan. Learn more here https://openweathermap.org/price. If you have a valid subscription to the One Call by Call plan, but still receive this error, then please see https://openweathermap.org/faq#error401 for more info..
```

So no more free lunch?

**Pull request merged**